config/passport.js

```
@@ -0,0 +1,905 @@
+    let user = await User.findById(req.user._id);
+    if (!user) {
+      // If user is not found in DB, use the one from the request because we are creating a new user
+      user = req.user;
+    }
+    const providerToken = user.tokens.find((token) => token.kind === providerName);
+    if (providerToken) {
+      providerToken.accessToken = accessToken;
+      if (accessTokenExpiration) {
+        providerToken.accessTokenExpires = moment().add(accessTokenExpiration, 'seconds').format();
+      } else {
+        delete providerToken.accessTokenExpires;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)